cocos/ui/UIEditBox/UIEditBox.h

```diff
@@ -407,7 +407,20 @@ namespace ui {
          * @param returnType One of the EditBox::KeyboardReturnType constants.
          */
         void setReturnType(EditBox::KeyboardReturnType returnType);
-            
+        
+        /**
+         * Set the inner padding in edit box.
+         * @param padding left, top, right, bottom.
+         */
+        void setPadding(float left, float top, float right, float bottom);
+        
+        /**
+         * Set the input text horizontal alignment and vertical alignment.
+         * @param hAlign the alignment for horizontally.
+         * @param vAlign the alignment for vertically.
+         */
+        void setTextAlignment(TextHAlignment hAlign, TextVAlignment vAlign);
```

Comment:
@anjandaffo 
I'm with @Mazyod , it's better to have getters along with setters.
And for the textAlignment API, the label class give us a example.

```
    /** Sets the Label's text horizontal alignment.*/
    void setAlignment(TextHAlignment hAlignment) { setAlignment(hAlignment,_vAlignment);}

    /** Returns the Label's text horizontal alignment.*/
    TextHAlignment getTextAlignment() const { return _hAlignment;}

    /** Sets the Label's text alignment.*/
    void setAlignment(TextHAlignment hAlignment,TextVAlignment vAlignment);

    /** Sets the Label's text horizontal alignment.*/
    void setHorizontalAlignment(TextHAlignment hAlignment) { setAlignment(hAlignment,_vAlignment); }

    /** Returns the Label's text horizontal alignment.*/
    TextHAlignment getHorizontalAlignment() const { return _hAlignment; }

    /** Sets the Label's text vertical alignment.*/
    void setVerticalAlignment(TextVAlignment vAlignment) { setAlignment(_hAlignment,vAlignment); }

    /** Returns the Label's text vertical alignment.*/
    TextVAlignment getVerticalAlignment() const { return _vAlignment; }

```